openedx/features/pakx/lms/pakx_admin_app/views.py

```diff
@@ -21,21 +34,49 @@ class UserProfileViewSet(viewsets.ModelViewSet):
     OrderingFilter.ordering_fields = ('id', 'name', 'email', 'employee_id')
     ordering = ['-id']
 
+    def create(self, request, *args, **kwargs):
+        if self.create_request_is_valid(request):
+            return self.create_new_user(request)
```

Comment:
please use the serializer create function. Django use forms, use serializer in DRF